### PR TITLE
update pro tip to clarify how to configure zsh shell

### PR DESCRIPTION
**Pull request by geirman**
Thanks to @alex_frantic for helping me solve this problem...
https://twitter.com/alex_frantic/status/657276624220659712

### CODE REVIEW

**Comment by pcottle** [comment]
Ideally we would have a more formalized mapping of `$SHELL` to rc file, but I guess this is fine now. Also it does make it sound like zsh and bash are the only shells out there (hopefully fish and csh users dont get offended) but no point in having perfect being the enemy of good

@facebook-github-bot import

**Comment by pcottle** [comment]
@facebook-github-bot shipit

**Comment by pcottle** [comment]
Oh whoops, we need you to sign the CLA first :P after that it can go in. sorry for the friction / annoyance

**Comment by pcottle** [comment]
@facebook-github-bot shipit

File: local-cli/server/util/launchEditor.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ function printInstructions(title) {
     '  stack frame to jump to the source file. The packager will launch your ',
     '  editor of choice. It will first look at REACT_EDITOR environment ',
     '  variable, then at EDITOR. To set it up, you can add something like ',
-    '  REACT_EDITOR=atom to your .bashrc.',
+    '  REACT_EDITOR=atom to your ~/.bashrc or export REACT_EDITOR=atom to your',
+    '  ~/.zshrc depending on which shell you use.',
```
Comment (frantic) [inline local-cli/server/util/launchEditor.js]: I'm sure we should use `export REACT_EDITOR=atom` even for bash.
A very cool DX detail could be figuring out what shell you use from `process.env` and suggesting relevant rc file.

**Comment by geirman** [comment]
totally confused why facebook-github-bot thinks I've updated this pull request

**Pull request closed without merging**